applications/test_wait_queue/src/lib.rs

```diff
@@ -55,7 +55,7 @@ fn rmain() -> Result<(), &'static str> {
     let t3 = spawn::new_task_builder(
         |tref| {
             warn!("DeezNutz:  testing spurious wakeup on task {:?}", tref);
```

Comment:
lol can't believe I left that in. Feel free to remove haha

Reply:
I like it. It's like a historical artifcat.